data-plane/config/channelv2/500-dispatcher.yaml

```diff
@@ -23,6 +23,7 @@ metadata:
     app: kafka-channel-dispatcher
     kafka.eventing.knative.dev/release: devel
 spec:
+  replicas: 0
```

Comment:
Is it temporary? perhaps can be removed before merging?

Reply:
Wdyt about initializing dispatcher pods with 0 replicas so that we leave it to the scheduler's autoscaler to decide on the initial scaling value? Otherwise the [default is 1](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#replicas) and most of the times the installation has to wait till the dispatcher pod terminates when there's no kafkasources installed....

Reply:
@pierDipi